marlowe-cli/examples/swap/run-swap.sh

```diff
@@ -346,42 +346,56 @@ cardano-cli query utxo --testnet-magic "$MAGIC" --address "$PARTY_B_ADDRESS" | s
 
 echo "## Clean Up"
 
-BURN_ADDRESS=addr_test1vqxdw4rlu6krp9fwgwcnld6y84wdahg585vrdy67n5urp9qyts0y7
+echo "Burning tokens issued of PARTY_A:"
 
-TX_CLEANUP_TOKEN_B=$(
-marlowe-cli util select --testnet-magic "$MAGIC"                   \
-                        --socket-path "$CARDANO_NODE_SOCKET_PATH"  \
-                        --asset-only "$TOKEN_B" "$PARTY_A_ADDRESS" \
-| sed -e 's/^TxIn "\(.*\)" (TxIx \(.*\))$/\1#\2/'                  \
-)
+marlowe-cli -- util burn --testnet-magic "$MAGIC" \
+                         --socket-path "$CARDANO_NODE_SOCKET_PATH" \
```

Comment:
@ladamesny we should probably unify the `mint` command options with this API so instead of having (in `mint`): `--required-signer` and `--change-address` we should have a single one `--issuer` which accepts both pieces at once.